lib/teleterm/api/proto/v1/service.proto

```diff
@@ -30,8 +30,10 @@ option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
 // TerminalService describes Teleterm service
 service TerminalService {
   // ListRootClusters lists root clusters
+  // Does not include detailed cluster information that would require a network request.
   rpc ListRootClusters(ListClustersRequest) returns (ListClustersResponse);
   // ListLeafClusters lists leaf clusters
+  // Does not include detailed cluster information that would require a network request.
```

Comment:
Could you add a comment to `GetCluster` that it makes a network request and returns detailed cluster information?